data.yml

```diff
@@ -191,6 +191,13 @@ landscape:
             homepage_url: https://palqee.ai
             logo: palqeepam.svg
             name: Palqee PAM
+          - item:
```

Comment:
Hi, thank you for the contribution. But I am not sure if we should put it in the evaluation category. How about AI in DB section?

Reply:
Thanks for the guidance.